### ISSUE TITLE
Possible to run example on Windows?

### ISSUE THREAD

**Issue by sendele** [tes/compoxure#76]
Being too lazy to install Redis, I removed the sections `"circuitbreaker"` and `"cache"` from `compoxure-1.0.5/example/config.json`.

But when hitting http://localhost:5000/ I get an empty response (Status Code: 200, but Content-Length: 0).

Is there a possibility to run the example without Redis at all?
Or am I missing something like a URL path or parameters?

**Comment by gabceb** [comment]
As per the Readme section 'To disable caching simply delete the entire config section'. Doesnt that work? Note that you still prob need the redis npm package dependency but redis should not be used. You can also use noCache on backends but we will probably try to connect to Redis initially

**Comment by sendele** [comment]
Thanks for your lightning-quick response! The initial call to Redis seems to be the exact problem:
```
Example compoxure server on http://localhost:5000
Example backend server on http://localhost:5001
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379
    at RedisClient.on_error (C:\workspace-luna\evaluation\compoxure-1.0.5\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (C:\workspace-luna\evaluation\compoxure-1.0.5\node_modules\redis\index.js:106:14)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at emitErrorNT (net.js:1253:8)
    at doNTCallback2 (node.js:452:9)
    at process._tickCallback (node.js:366:17)
```

Deleting the entire `cache` config section is exactly what I did. More formally, I replaced
```
    [...]
    "circuitbreaker":{
        "windowDuration":10000,
        "numBuckets": 10,
        "errorThreshold": 20,
        "volumeThreshold": 5,
        "includePath":true,
        "publishToRedis":"redis://localhost:6379?db=0"
    },
    "cache": {
        "engine": "redis",
        "url":"redis://localhost:6379?db=0",
        "apiEnabled": true
    },
    "hogan": {
        "delimiters": "{{ }}"
    }
}
```
by
```
    [...]
    "circuitbreaker":{
        "windowDuration":10000,
        "numBuckets": 10,
        "errorThreshold": 20,
        "volumeThreshold": 5,
        "includePath":true,
        "publishToRedis":"redis://localhost:6379?db=0"
    },
    "hogan": {
        "delimiters": "{{ }}"
    }
}
```

I tried an additional `"noCache": true,` (is that correct?) on all backends, but still the response is empty.

**Comment by sendele** [comment]
Ok, I've overcome my laziness and installed Redis, more precisely https://github.com/MSOpenTech/redis as I'm working on Windows.
After resetting my `config.json`, the error on startup disappeared, but still I get the same result (empty response).

**Comment by cliftonc** [comment]
This may be fixed with the merge of the fix for #75 - can you try now?

**Comment by sendele** [comment]
I tried Compoxure 1.0.7 and works now. Seems like this is a duplicate of  #75
